string csvName
string datasetName
---
bool success